.github/workflows/test_prettier.yml

```diff
@@ -1,14 +1,17 @@
 # See: https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions
 
-name: 'Test Documentation'
+name: 'Test Prettier'
 
 on:
   workflow_dispatch:
   pull_request:
     branches:
       - main
     paths:
-      - '**/*.md'
+      - '**/*.{md,json,yml,scss}'
+
+env:
+  HUSKY: 0
```

Comment:
This is the only workflow that really needs this, right? Because this runs `npm install` in the root?

Reply:
Exactamundo